dependencies.props

```diff
@@ -59,6 +60,7 @@
     <TestILCArmretPackageVersion>$(ProjectNTfsTestILCPackageVersion)</TestILCArmretPackageVersion>
     <TestILCX86retPackageVersion>$(ProjectNTfsTestILCPackageVersion)</TestILCX86retPackageVersion>
     <OptimizationDataVersion>2.0.0-rc-61101-17</OptimizationDataVersion>
+    <CoreFxOptimizationDataVersion>99.99.99-master-20180817-0151</CoreFxOptimizationDataVersion>
```

Comment:
Let's make sure this and the CurrentRef are updated to what is current in dotnet/version